components/greaselion/browser/greaselion_download_service.cc

```diff
@@ -212,11 +217,16 @@ void GreaselionDownloadService::OnDATFileDataReady(std::string contents) {
     rule_dict->GetList(kScripts, &scripts_value);
     const std::string* run_at_ptr = rule_it.FindStringPath(kRunAt);
     const std::string run_at_value = run_at_ptr ? *run_at_ptr : "";
+    const std::string* messages_ptr = rule_it.FindStringPath(kMessages);
+    base::FilePath messages_value;
+    if (messages_ptr) {
+      messages_value = base::FilePath::FromUTF8Unsafe(messages_ptr->c_str());
```

Comment:
According to https://developer.chrome.com/extensions/i18n the subdirectory must always be named `_locales` so let's not specify it in the JSON file, which would also avoid the entire issue of having part of a file path as a string in a JSON file. A boolean value (has l18n yes/no) will suffice.

Reply:
Makes sense. You good with that change @petemill?

Reply:
Spoke with @petemill and we'll keep this as-is for now, since different extensions have different localization needs.